internal/pubsub/gcppubsub/gcppubsub.go

```diff
@@ -51,18 +51,18 @@ func OpenTopic(ctx context.Context, client *raw.PublisherClient, proj gcp.Projec
 // openTopic returns the driver for OpenTopic. This function exists so the test
 // harness can get the driver interface implementation if it needs to.
 func openTopic(ctx context.Context, client *raw.PublisherClient, proj gcp.ProjectID, topicName string) (driver.Topic, error) {
+	path := fmt.Sprintf("projects/%s/topics/%s", proj, topicName)
 	client, err := raw.NewPublisherClient(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("gcppubsub: creating publisher client: %v", err)
 	}
-	ok, err := topicExists(ctx, client, topicName)
+	ok, err := topicExists(ctx, client, path)
 	if err != nil {
 		return nil, fmt.Errorf("gcppubsub: checking existence of topic: %v", err)
 	}
 	if !ok {
 		return nil, fmt.Errorf("gcppubsub: topic named %q does not exist", topicName)
 	}
-	path := fmt.Sprintf("projects/%s/topics/%s", proj, topicName)
 	return &topic{path, client}, nil
 }
 
```

Comment:
Should be topicPath now, right? It's not just the name.

Reply:
What should be topicPath? Are you saying to run s/path/topicPath/?

Reply:
Sorry, I'm referring to line 69, `func topicExists` which takes a `topicName string` argument. IIUC, that is not a topic name, it's a path. It will be easy to mix them up, so be consistent with the names. I don't care about `topicPath` vs `path`.

Reply:
Done.